contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,726 @@
+---
+title: "The most popular PostHog alternatives, compared"
+date: 2023-06-22
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+As an all-in-one platform, PostHog replaces _a lot_ of legacy tools. PostHog is great if you:
+
+1. Need quantitative _and_ qualitative insights into user behavior
+2. Want to use feature flags for rolling out new features, kill switches, and feature gating
+3. Desire one integrated tool over a complex stack of discrete solutions
+
+This guide is an overview of some popular alternatives to PostHog, how they compare, and [why people choose PostHog](#why-people-choose-posthog).
+
+> **Note:** _User reviews and comments are hand curated from G2.com_
```

Comment:
instead of including the sentence, maybe we can just hyperlink to the reviews to their G2 reviews?

Reply:
No. Don't want people leaving the site, plus it's a pita to go find them all.